rest/route/project.go

```diff
@@ -1121,7 +1122,7 @@ func (h *projectVarsPutHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(err,
 			"error updating projects with matching keys"))
 	}
-	_, err = artifact.RotateSecrets(h.replaceVars.ToReplace, h.replaceVars.Replacement, h.replaceVars.DryRun)
```

Comment:
I think this isn't right; we just shouldn't call RotateSecrets at all if RotateFiles isn't true, but RotateSecrets should still consider DryRun.